src/App/Lobby/LobbyService.ts

```diff
@@ -0,0 +1,12 @@
+import AddressHelper from '../AddressHelper';
+import request from 'superagent';
+
+export class LobbyService {
+  public static async newRoom(gameCode: string): Promise<string> {
+    const response = await request
+      .post(`http://${AddressHelper.getServerAddress()}/games/${gameCode}/create`)
```

Comment:
Putting http:// here  is problematic because ```AddressHelper.getServerAddress()``` might specify a protocol.
https://github.com/freeboardgame/FreeBoardGame.org/blob/6a8ec3966034cefb2bc15d10a4da058304e6da73/src/App/AddressHelper.ts#L5

In our production environment, ```process.env.BGIO_SERVER_URL=https://freeboardgame.org```, so this would be ```http://https://freeboardgame.org/games...```

I removed the http:// from this and changed getServerAddress() to always return a protocol.

Reply:
I think I initially coded and I assumed that getServerAddress would always return host:port. So we found a bug in the original code...

Reply:
Well, kinda.  BGIO would auto-insert the http:// -- it probably not the best idea for me to rely on BGIO to do this, but fixed now :)